cobigen/cobigen-core/src/main/java/com/devonfw/cobigen/impl/config/reader/ContextConfigurationReader.java

```diff
@@ -193,6 +169,12 @@ public void readConfiguration() {
     }
   }
 
+  /**
+   * Checks if a conflict with the old and modular configuration exists
+   *
+   * @param configRoot Path to root directory of the configuration
+   * @param contextFile Path to context file of the configuration
+   */
```

Comment:
This whole method and case can be removed as it was only intended for the temporary solution e.g. template.xml and context.xml in the same folder.
This method just ensured that if another context.xml was found in the root, CobiGen stopped instead of crashing.